linera-views-derive/src/lib.rs

```diff
@@ -223,22 +198,18 @@ fn generate_view_code(input: ItemStruct, root: bool) -> TokenStream2 {
 }
 
 fn generate_save_delete_view_code(input: ItemStruct) -> TokenStream2 {
```

Comment:
The term `generate_save_delete_view_code` is inherited from the first version where we had a `delete` functionality. This is no longer the case.
So, maybe rename the function to `generate_rootview_code`.

Reply:
https://github.com/linera-io/linera-protocol/pull/3854/commits/4b4ef12d18e22db810555a52ec405ecbf95fde85